web_client/views/imageViewerWidget/leaflet.js

```diff
@@ -4,8 +4,15 @@ var LeafletImageViewerWidget = ImageViewerWidget.extend({
     initialize: function (settings) {
         ImageViewerWidget.prototype.initialize.call(this, settings);
 
-        $('head').prepend(
-            $('<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.7/leaflet.css">'));
+        if (!$('#large_image-leaflet-css', 'head').length) {
```

Comment:
Nope. That's an artifact of me thinking about doing the lookup of `<head>` only once, but then removing it. I'll simplify it further now.